docs/guides/python_chip_controller_building.md

```diff
@@ -217,84 +166,74 @@ with network credentials.
 
-    ```
-    chip-device-ctrl > zcl NetworkCommissioning AddOrUpdateWiFiNetwork 1234 0 0 ssid=str:TESTSSID credentials=str:P455W4RD breadcrumb=0
-    ```
-
-2. Enable the Wi-Fi interface on the device by executing the following command:
-
-    ```
-    chip-device-ctrl > zcl NetworkCommissioning ConnectNetwork 1234 0 0 networkID=str:TESTSSID breadcrumb=0
-    ```
+```
+chip-device-ctrl > set-pairing-wifi-credential TESTSSID P455W4RD
+```
 
-### Step 6: Close Bluetooth LE connection.
+### Step 5: Commission the Matter accessory device over Bluetooth LE
 
-After the Matter accessory device was provisioned with Thread or Wi-Fi
-credentials (or both), the commissioning process is finished. The Python CHIP
-controller is now using only the IPv6 traffic to reach the device, so you can
-close the Bluetooth LE connection. To close the connection, run the following
-command:
+The controller uses a 12-bit value called **discriminator** to discern between
+multiple commissionable device advertisements. Moreover, a 27-bit **PIN code**
+is used by the controller to authenticate in the device. You can find those
+values in the logging terminal of the device (for example, UART). For example:
 
 ```
-chip-device-ctrl > close-ble
+I: 254 [DL]Device Configuration:
+I: 257 [DL] Serial Number: TEST_SN
+I: 260 [DL] Vendor Id: 9050 (0x235A)
+I: 263 [DL] Product Id: 20043 (0x4E4B)
+I: 267 [DL] Hardware Version: 1
+I: 270 [DL] Setup Pin Code: 20202021
+I: 273 [DL] Setup Discriminator: 3840 (0xF00)
+I: 278 [DL] Manufacturing Date: (not set)
+I: 281 [DL] Device Type: 65535 (0xFFFF)
 ```
 
-### Step 7: Discover IPv6 address of the Matter accessory.
-
-The Matter controller must discover the IPv6 address of the node that it
-previously commissioned. Depending on the network type:
-
--   For Thread, the Matter accessory uses SRP (Service Registration Protocol) to
-    register its presence on the Thread Border Router’s SRP Server.
--   For Wi-Fi or Ethernet devices, the Matter accessory uses the mDNS (Multicast
-    Domain Name System) protocol.
+Run the following command to establish the secure connection over Bluetooth LE,
+with the following assumptions for the Matter accessory device:
 
-Assuming your Node ID is _1234_ (use the Node ID you noted down when you
-established the secure connection over Bluetooth LE)), run the following
-command:
+-   The discriminator of the device is _3840_
+-   The setup pin code of the device is _20202021_
+-   The temporary Node ID is _1234_
 
 ```
-chip-device-ctrl > resolve 1234
+chip-device-ctrl > connect -ble 3840 20202021 1234
 ```
 
-A notification in the log indicates that the node address has been updated. The
-IPv6 address of the device is cached in the controller for later usage.
+You can skip the last parameter, that is the Node ID. If you skip it, the
+controller will assign it randomly. However, note the Node ID down, because it
```

Comment:
```suggestion
You can skip the last parameter, the Node ID, in the command. If you skip it, the
controller will assign it randomly. In that case, note down the Node ID, because it
```